src/EventHub/EventHub.Autorest/README.md

```diff
@@ -343,6 +343,14 @@ directive:
     set:
       property-name: MessageRetentionInDays
 
+  - where:
+      verb: New
+      subject: EventHub
+      parameter-name: MessageRetentionInDays
+    set:
+      breaking-change:
+        change-description: -MessageRetentionInDays would be deprecated and would be replaced by -RetentionTimeInHours
```

Comment:
I think we can keep deprecated-by-version: 4.0.0. For Az.EventHub, we can expect a major version change